tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_training_cuda10_2

```diff
@@ -181,10 +174,9 @@ ENV SSL_CERT_FILE=/opt/_internal/certs.pem
 CMD ["/bin/bash"]
 
 #Build manylinux2014 docker image end
-
-ARG PYTHON_VERSION=3.6
+ARG PYTHON_VERSION=3.9
 ARG TORCH_VERSION=1.11.0
-ARG OPSET_VERSION=12
+ARG OPSET_VERSION=14
```

Comment:
I understand this is related to the ONNX opset version but what does this really control? Is this related to what we test in the pipeline?